pulp_rpm/tests/functional/api/test_retention_policy.py

```diff
@@ -0,0 +1,144 @@
+# coding=utf-8
+"""Tests for the retention policy feature of repositories."""
+
+from collections import defaultdict
+
+from pulp_smash import config
+from pulp_smash.pulp3.utils import (
+    gen_repo,
+    get_added_content_summary,
+    get_content,
+    get_content_summary,
+    get_removed_content,
+    get_removed_content_summary,
+    delete_orphans,
+)
+
+from pulp_rpm.tests.functional.constants import (
+    PULP_TYPE_PACKAGE,
+    RPM_FIXTURE_SUMMARY,
+)
+from pulp_rpm.tests.functional.utils import (
+    gen_rpm_client,
+    gen_rpm_remote,
+    monitor_task,
+    progress_reports,
+)
+from pulp_rpm.tests.functional.utils import set_up_module as setUpModule  # noqa:F401
+from pulp_rpm.tests.functional.utils import PulpTestCase
+
+from pulpcore.client.pulp_rpm import (
+    RepositoriesRpmApi,
+    RpmRepositorySyncURL,
+    RemotesRpmApi,
+)
+
+
+class RetentionPolicyTestCase(PulpTestCase):
```

Comment:
I can rename this too but the name still seems kind of fitting when used as a noun.